### DISCUSSION TITLE
unRAID users - beware of 6.12 btrfs issues on your cache drive

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#73] [General]
I have been running unRAID without any issues.  A few days ago I upgraded from 6.11.5 to 6.12.4 and I am getting major issues that reoccur after a few hours.  It seems that a bunch of other folks are seeing similar problems after upgrading and the issues go away when you revert back to 6.11.5.

For more detail see this:  https://forums.unraid.net/topic/141065-btrfs-error-and-read-only-cache-since-updating-to-612/

**Comment by wayner9** [comment]
I think you are good as I think this occurred with any 6.12 version.  I rolled back to 6.11.5, reformatted my cache drive and was fine for three days.  I upgraded again last night and the corruption issue reoccurred within 8 hours.

**Comment by MechlingBurgh** [comment]
Why are you guys using Btrfs on the cache drives what would be the advantage of doing that?

**Reply by wayner9** [reply]
I don't remember consciously choosing BTRFS, that must have been the default when I built this system in Dec 2021.

Last night I switched the drive to XFS, and so far, so good.  But I am going to add a second cache drive and mirror them, and to do that XFS won't work so I will go to ZFS.

My cache drive is a 500GB NVME and they now sell for C$35, so since I have a second NVME slot on my mobo it seems like a no-brainer to have a mirrored cache drive.  Then I don't have to worry about stuff like doing backups of the appdata or other shares.

**Reply by wayner9** [reply]
I just checked and my original unRAID server, which I believe is from 2017, also has a BTRFS cache.  Again, I don't remember consciously deciding to use that file system type.

**Reply by wayner9** [reply]
Were you having issues with BTRFS?  I have bought a second drive to mirror my cache, but I haven't got around to adding it yet.  It is just setting on top of my server.  Any tricks to adding a second drive to your cache?  I have always run with only one cache, until now.
